drivers/hmis/app/jobs/hmis/auto_exit_job.rb

```diff
@@ -0,0 +1,84 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Hmis
+  class AutoExitJob < BaseJob
+    def perform
+      Hmis::Hud::Project.hmis.each do |project|
+        config = Hmis::AutoExitConfig.config_for_project(project)
+        next unless config.present?
+        raise "Auto-exit config unusually low: #{config.length_of_absence_days}" if config.length_of_absence_days < 30
+
+        project.enrollments.open_excluding_wip.each do |enrollment|
+          most_recent_contact = if project.project_type == 1 # Night-by-night Emergency Shelter
```

Comment:
maybe (or do we not have that on the HMIS side?
```suggestion
          most_recent_contact = if project.es_nbn?
```

Reply:
We don't seem to have this on the HNMI side. I tried `Hmis::Hud::Project.last.es_nbn?` and it gave me a no method error IIRC

Reply:
Huh, @gigxz any objection to putting something like the following into `Hmis::Hud::Project`?  (We can simplify it because you just use the `project_type` column on the HMIS side, no overrides.
```
HudUtility2024.residential_project_type_numbers_by_code.each do |k, v|
      scope k, -> { where(project_type_column => v) }
      define_method "#{k}?" do
        v.include? project_type_to_use
      end
    end
```

Reply:
Yeah lets do it

Reply:
@gigxz This is done